db-scripts/src/main/resources/cgds.sql

```diff
@@ -775,6 +791,22 @@ CREATE TABLE `clinical_event_data` (
 );
 
 -- --------------------------------------------------------
+DROP TABLE IF EXISTS `reference_genome_gene`;
+CREATE TABLE `reference_genome_gene` (
+    `ENTREZ_GENE_ID` int(11) NOT NULL,
+    `CYTOBAND` varchar(64) NOT NULL,
```

Comment:
Should this cytoband be the same length as cytoband in the gene table? Is cytoband in the gene table going to be removed in a future PR?

Reply:
cytogenetic bands are different from different genome builds. cytoband in the gene table will be removed in a future PR.

Reply:
line 794 also removed, duplicated with line 86. Nice catch!

Reply:
changed the length of CYTOBAND in gene table to varchar(64) to be consist, even this field will be removed in a future PR.